lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -1017,18 +1061,22 @@ class DecomposeAtenNativeBatchNormOp
     // 3. output = normalizedInput * weight + bias
     Value batchNormOutput = normalizedInput;
     if (!weight.getType().isa<Torch::NoneType>()) {
```

Comment:
Should we also make the prototype a bit simpler? like `static bool isNone(Value v)`? 
As it will give the caller an option whether you want to go ahead with none value or return failure in the case of none value.

Reply:
This can be handled later.